src/pages/questionPage.js

```diff
@@ -52,26 +52,36 @@ export const initQuestionPage = () => {
 };
 
 const nextQuestion = () => {
-  const correctAnswer = quizData.questions[quizData.currentQuestionIndex].correct;
-  const addClass = quizData.currentQuestionAnswer === correctAnswer ? 'correct' : 'wrong';
+  addAnswerToStorage(quizData.currentQuestionAnswer);
+  const correctAnswer =
+    quizData.questions[quizData.currentQuestionIndex].correct;
+  const addClass =
+    quizData.currentQuestionAnswer === correctAnswer ? 'correct' : 'wrong';
   const body = document.getElementById(USER_INTERFACE_ID);
-  if (quizData.currentQuestionAnswer===null) {
+
+  if (quizData.currentQuestionAnswer === null) {
     const alertElement = createAlertElement();
     body.appendChild(alertElement);
-    quizData.currentQuestionIndex = quizData.currentQuestionIndex -1;
-  } else if(quizData.currentQuestionAnswer === correctAnswer ) {
-    currentAnswerElement.classList.remove('selected')
-    currentAnswerElement.classList.add(addClass)
+    quizData.currentQuestionIndex = quizData.currentQuestionIndex - 1;
+  } else if (quizData.currentQuestionAnswer === correctAnswer) {
+    currentAnswerElement.classList.remove('selected');
+    currentAnswerElement.classList.add(addClass);
   } else {
-    currentAnswerElement.classList.remove('selected')
-    currentAnswerElement.classList.add(addClass)
+    currentAnswerElement.classList.remove('selected');
+    currentAnswerElement.classList.add(addClass);
   }
-  quizData.currentQuestionAnswer = null
+  quizData.currentQuestionAnswer = null;
   quizData.currentQuestionIndex = quizData.currentQuestionIndex + 1;
-  document.getElementById(NEXT_QUESTION_BUTTON_ID).removeEventListener('click', nextQuestion);
+  document
+    .getElementById(NEXT_QUESTION_BUTTON_ID)
+    .removeEventListener('click', nextQuestion);
+
+  quizData.currentQuestionIndex < quizData.questions.length - 1
```

Comment:
This line is a bit unclear, can you add a comment explaining what it does and why it's there? So that other reading the code can understand it more easily?